src/parser_error.rs

```diff
@@ -82,7 +82,7 @@ impl QueryParserErrorIntoPy for tv::query::QueryParserError {
 }
 
 /// Error in the query syntax.
-#[pyclass(frozen)]
+#[pyclass(frozen, module = "tantivy.tantivy")]
```

Comment:
I am not sure whether these changes are required, since these classes are added with `m.add_class` in https://github.com/quickwit-oss/tantivy-py/blob/master/src/lib.rs#L122.